swupd/fullfiles_test.go

```diff
@@ -65,23 +112,84 @@ func TestCreateFullfiles(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// All correct files were created.
 	for _, desc := range files {
-		tarName := filepath.Join(outputDir, desc.hash+".tar")
-		if desc.version != m.Header.Version {
-			mustNotExist(t, tarName)
-		} else {
-			mustExist(t, tarName)
+		tarName := filepath.Join(outputDir, desc.ExpectedHash+".tar")
+		if desc.Version != m.Header.Version {
```

Comment:
optional: move this before doing the string work?

Reply:
Done.